frontend/src/components/LandingPage/Explanation/Explanation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Jumbotron } from 'reactstrap'
+//Import components that you create here
```

Comment:
Is this a good comment to have here?

Reply:
Oops, forgot to remove some comments. Will try to add some of my own instead.